pkg/clients/connection/connection.go

```diff
@@ -53,9 +53,9 @@ func FromParameters(p v1alpha3.ConnectionParameters) *servicenetworking.Connecti
 	}
 }
 
-// UpToDate returns true if the observed Connection is up to date with the
+// IsUpToDate returns true if the observed Connection is up to date with the
 // supplied ConnectionParameters.
-func UpToDate(p v1alpha3.ConnectionParameters, observed *servicenetworking.Connection) bool {
+func IsUpToDate(p v1beta1.ConnectionParameters, observed *servicenetworking.Connection) bool {
```

Comment:
We might be able to update this bespoke logic to use `cmp` like our other `IsUpToDate` functions.

Reply:
We could replace the range stanza, but the sorting and length comparison will still be needed (well we technically don't have to compare length but should because no point in sorting if they are a different length).

Reply:
cmp has functionality to do this - https://godoc.org/github.com/google/go-cmp/cmp/cmpopts#SortSlices.

I believe this function is equivalent to something like:

```go
return cmp.Equal(p.ReservedPeeringRanges, observed.ReservedPeeringRanges cmpopts.SortSlices(func(i, j string) bool { return i < j }))
```